apps/laboratory/src/pages/library/multichain-basic.tsx

```diff
@@ -7,10 +7,23 @@ import { UpaTests } from '../../components/UPA/UpaTests'
 import { ConstantsUtil } from '../../utils/ConstantsUtil'
 import { ThemeStore } from '../../utils/StoreUtil'
 
-const networks = ConstantsUtil.AllNetworks
+const networks = [
+  ...ConstantsUtil.AllNetworks,
+  defineChain({
+    id: '91b171bb158e2d3848fa23a9f1c25182',
+    name: 'Polkadot',
+    network: 'polkadot',
+    nativeCurrency: { name: 'Polkadot', symbol: 'DOT', decimals: 18 },
+    rpcUrls: {
+      default: { http: ['https://rpc.polkadot.io'] }
+    },
+    chainNamespace: 'polkadot',
+    caipNetworkId: 'polkadot:mainnet'
+  })
+]
 
 const modal = createAppKit({
-  networks,
+  networks: networks as any,
```

Comment:
Can we use instead any?
```suggestion
  networks: networks as [AppKitNetwork, ...AppKitNetwork[]],
```